File: Programmers-Level2/카카오 프렌즈 컬러링북/Main.java
public class Main {
  public static void main(String[] args) {
    int picture[][] = {{1, 1, 1, 0}, {1, 2, 2, 0}, {1, 0, 0, 1}, {0, 0, 0, 1}, {0, 0, 0, 3}, {0, 0, 0, 3}};
    
    Solution s = new Solution();
    print_solution(s.solution(6, 4, picture));
  }

  public static void print_solution(int[] res){
    System.out.print("[");

    for(int i = 0; i < res.length; i++){
      System.out.print(res[i]);

      if((i + 1) != res.length) System.out.print(", ");
    }

    System.out.println("]");
  }
}

File: Programmers-Level2/카카오 프렌즈 컬러링북/main.cpp
#include "solution.h"

void print_solution(vector<int> res);

int main() {
  print_solution(solution(6, 4, {{1, 1, 1, 0}, {1, 2, 2, 0}, {1, 0, 0, 1}, {0, 0, 0, 1}, {0, 0, 0, 3}, {0, 0, 0, 3}}));

  return 0;
}

void print_solution(vector<int> res) {
  cout << "[";

  for (int i = 0; i < res.size(); i++) {
    cout << res[i];

    if ((i + 1) != res.size()) cout << ", ";
  }

  cout << "]";
}